@xen-orchestra/web/src/components/hosts/card/HostHardwareSpecificationsCard.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <UiCard>
+    <UiCardTitle>
+      {{ t('hardware-specifications') }}
+    </UiCardTitle>
+    <div class="content">
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('manufacturer-info') }}</template>
+        <template v-if="manufacturerInfo" #value>{{ manufacturerInfo }}</template>
+        <template v-if="manufacturerInfo" #addons>
+          <VtsCopyButton :value="manufacturerInfo" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('core-socket') }}</template>
+        <template #value>{{ `${host.cpus.cores} (${host.cpus.sockets})` }}</template>
+        <template #addons>
+          <VtsCopyButton :value="`${host.cpus.cores} (${host.cpus.sockets})`" />
+        </template>
+      </VtsCardRowKeyValue>
```

Comment:
```suggestion
      <VtsCardRowKeyValue>
        <template #key>{{ t('core-socket') }}</template>
        <template v-if="coreSocketInfo" #value>{{ coreSocketInfo }}</template>
        <template v-if="coreSocketInfo" #addons>
          <VtsCopyButton :value="coreSocketInfo" />
        </template>
      </VtsCardRowKeyValue>
```

Maybe we can simplify a bit more like that ?